tests/unit/ReferenceListTest.php

```diff
@@ -52,7 +52,7 @@ public function getConstructorArg() {
 	public function testCanConstructWithReferenceListObject() {
 		$original = new ReferenceList( array( new Reference( array( new PropertyNoValueSnak( 1 ) ) ) ) );
 		$copy = new ReferenceList( $original );
-		$this->assertSame( 1, $copy->count() );
+		$this->assertEquals( $original, $copy );
```

Comment:
Define "sane". PHPUnit does not add much magic on top of PHP's. It delegates the object comparison to PHP. We already run into horrifying problems when comparing list objects that extend one of PHP's classes. I'm very alarmed whenever I see something like this. I will not merge this PR.